src/rpc/rpcquorums.cpp

```diff
@@ -360,7 +363,17 @@ UniValue quorum_sigs_cmd(const JSONRPCRequest& request)
     uint256 msgHash = ParseHashV(request.params[3], "msgHash");
 
     if (cmd == "sign") {
-        return llmq::quorumSigningManager->AsyncSignIfMember(llmqType, id, msgHash);
+        if(request.params.size() > 4){
+            uint256 quorumHash = ParseHashV(request.params[4], "quorumHash");
+            auto quorum = llmq::quorumManager->GetQuorum(llmqType, quorumHash);
+            if (!quorum) {
+                throw JSONRPCError(RPC_INVALID_PARAMETER, "quorum not found");
+            }
+            llmq::quorumSigSharesManager->AsyncSign(quorum, id, msgHash);
+            return true;
+        }else{
```

Comment:
Nit: Same here, this else could even be omitted and just out-dent the next line since the `if` block above it returns.